frontend/src/types/crawler.ts

```diff
@@ -126,7 +126,7 @@ export type Crawl = CrawlConfig & {
   firstSeed: string;
   seedCount: number;
   stopping: boolean;
-  collections: string[];
```

Comment:
If we're going to be fetching names in `collections`, should that still be part of the type here, just with a different type signature, similar to resources?

Reply:
Or maybe Sua will add that when we make use of the new collection names?

Reply:
yea, was thinking that'd be added as part of the frontend PR that is using them